### DISCUSSION TITLE
Log Filters

### DISCUSSION THREAD

**Discussion by glassmancody** [mbround18/valheim-docker#646] [Ideas]
Fantastic setup! My only gripe is there is no builtin way to filter logs. I routinely am grepping through logs with `docker-compose logs ...`  but it's filled with warnings which I know can be ignored.

A similar project has environment variables for log filters, and the default settings help a lot with getting rid of garbage message spam (https://github.com/lloesche/valheim-server-docker#log-filters). 

Would this be something the project could adapt? Even a simple set of regex rules would be more then sufficient, nothing complex needed here.

**Comment by mbround18** [comment]
It uses https://GitHub.com/rust-cli/env_logger under the hood which has log filters. But I'm purely doing a pass through on game log filters. 

I could potentially look into designing a more robust logging system to parse and manage the game server logs. But that might be a good chunk of work, if it's a popular request feature I can drive into it :) but moving around atm so don't have access to a full machine
